abn/core/service.go

```diff
@@ -37,6 +37,24 @@ var (
 
 // Start is entry point to configure services and start them
 func Start() {
+	w := initializeServer()
```

Comment:
In `autox`, we structure the Start function like [this](https://github.com/iter8-tools/iter8/blob/3038ba07e857966209535b84f5736c33a39e74d5/autox/watcher.go#L20). 

This implies, we structure the `cmd` like [this](https://github.com/iter8-tools/iter8/blob/3038ba07e857966209535b84f5736c33a39e74d5/cmd/autox.go#L26)

More importantly, we were able to test it like [this](https://github.com/iter8-tools/iter8/blob/3038ba07e857966209535b84f5736c33a39e74d5/autox/watcher_test.go#L31) -- the key point here is how the test simply calls `Start(...)` and does not copy and paste the internal code from  the `Start(...)` func inside the test func.

Reply:
Please structure the `abn` Start(...) similar to the `autox` Start so that it can be tested the same way (without copying the internals of Start into the test func).

Reply:
much nicer pattern. thx.